contracts/token/PausableToken.sol

```diff
@@ -15,11 +15,22 @@ import '../lifecycle/Pausable.sol';
 
 contract PausableToken is Pausable, StandardToken {
 
+  /**
+  * @dev Transfer from one addres to another.
+  * @param _to address The addres to transfer to
+  * @param _value uint The value to transfer
+  */
```

Comment:
Do we really need to repeat the docs from StandardToken here? Can't we say something like `@see StandardToken#transfer` ?